mentoring/answer.py

```diff
@@ -24,39 +21,146 @@
 # Imports ###########################################################
 
 import logging
-
 from lazy import lazy
 
-from xblock.fragment import Fragment
-
-from .light_children import LightChild, Boolean, Scope, String
 from .models import Answer
-from .utils import render_template
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Float, Integer, String
+from xblock.fragment import Fragment
+from xblock.validation import ValidationMessage
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+from .step import StepMixin
+import uuid
 
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
+
 
+# Make '_' a no-op so we can scrape strings
+def _(text):
+    return text
 
 # Classes ###########################################################
 
-class AnswerBlock(LightChild):
+
+class AnswerMixin(object):
```

Comment:
I had actually tried decorating this mixin as you suggest, but it doesn't work, since it's not a real XBlock:
```
AttributeError: type object 'AnswerMixin' has no attribute '_services_requested'
```

So instead I had to stick with decorating each XBlock that uses this mixin.